docs/banzai/index.rst

```diff
@@ -2,80 +2,136 @@
-take a mean of the remaining pixels as usual.
 
-
-Individual Camera Considerations
---------------------------------
-The Spectral and SBIG cameras produce single extension fits files. The Sinistro frames, however, produce
-four amplifier outputs. Currently this data is saved as a data cube (but alternatively could be stored as
-a multi-extension fits file). Currently the fits-preprocessor subtracts the overscan, removes amplifier cross-talk,
-and mosaics the 4 amplifiers into a single image before the pipeline processes the data.The preprocessor will eventually be
-merged into this this pipeline.
-
-The other difficulty with the Sinistro camera is that
-the top amplifiers (3 and 4) have more light sensitive pixels than the bottom amplifiers (1 and 2). All of the parallel
-overscan rows are discarded. One edge row is discarded for the amplifiers that have an extra light sensitive row.
-The final frame size is 4096x4096.
-Currently only 1x1 binning is supported for Sinsitros.
-
-
-Bad Pixel Masks
----------------
-Currently bad pixel masks are not included, but will be soon. Using robust statistics should guard from the effects of
-bad pixels (mostly) so we still produce clean data products.
+Stages
+------
+BANZAI is comprised of different stages, each corresponding to a single reduction step, which are chained together
+to process images. The individual stages are described below, in the order that they are executed.
 
 
 Overscan
 ========
-At LCOGT, some of our cameras have overscan regions and others that do not. Of the currently deployed instruments,
-the Sinistro cameras and the Spectral cameras have overscan regions. The SBIG cameras are capable of producing an
-overscan region but are currently configured to not produce either parallel or serial overscan.
-
-We currently use the header the keyword BIASSEC to identify the overscan region. If this keyword is set to
+We currently use the header the keyword ``BIASSEC`` to identify the overscan region. If this keyword is set to
 "Unknown", then we simply skip subtracting the overscan. We estimate a single overscan value for the whole image
 (rather than row by row).
-This is saved in the header under the keyword 'BIASLVL'. If there is no overscan section, this value is derived
+This is saved in the header under the keyword ``BIASLVL``. If there is no overscan section, this value is derived
 from the average value from the bias frames.
 
-The Sinistro frames do have overscan regions, but the overscan is currently removed by the preprocessor.
-
 
 Crosstalk
 =========
-Currently, only Sinstro images are read out using multiple amplifiers. The Sinsitro frames do have significant
+Currently, only Sinistro images are read out using multiple amplifiers. The Sinistro frames do have significant
 crosstalk between amplifiers, but this is currently removed by the preprocessor. The crosstalk is removed using
```

Comment:
This is not true anymore actually. The preprocessor is no longer in use.

Reply:
Fixed